cmd/global_test.go

```diff
@@ -10,18 +10,18 @@ func Test(t *testing.T) {
 	TestingT(t)
 }
 
-var _ = Suite(&EtcdCtl{})
+var _ = Suite(&Global{})
 
-type EtcdCtl struct{}
+type Global struct{}
 
-func (s *EtcdCtl) TestBase64Encode(c *C) {
+func (s *Global) TestBase64Encode(c *C) {
 	c.Parallel()
 	expected := "SGVsbG8sV29ybGQ="
 	obtained := base64Encode("Hello,World")
 	c.Assert(obtained, Equals, expected)
 }
 
-func (s *EtcdCtl) TestBase64Decode(c *C) {
+func (s *Global) TestBase64Decode(c *C) {
```

Comment:
```
./tidb-ctl base64decode -v cAAAAAABX+s=
hex: 7000000000015feb
uint64: 8070450532248018923
```

adding test for this value is better.

Reply:
I add test in https://github.com/pingcap/tidb-ctl/blob/1dec121f921d1abae4dcaf0c4a9d2070ddaa895f/cmd/cmdctl_test.go#L41